contributed_definitions/NXxpcs.nxdl.xml

```diff
@@ -0,0 +1,596 @@
+
+            This format is chosen because results plotting packages are not common and simple I/O is required by end user.
+            The lists can be accessed as lists, arrays or via keys
+            </doc>
+          </field>
+          <field name="dynamic_phi_list" type="NX_NUMBER" units="NX_PER_LENGTH" minOccurs="0">
+            <doc>
+            Array of :math:`\varphi` value for each pixel.
+
+            List order is determined by the index value of the associated roi map starting at ``1``.
+            </doc>
+          </field>
+          <field name="static_roi_map" type="NX_DIMENSIONLESS" minOccurs="0">
+            <doc>
+            roi index array.
+
+            The values of this mask index the :math:`|Q|` value from the
+            the ``static_q_list`` field.
+
+            The ``units`` attribute should be set to ``"au"``
+            indicating arbitrary units.
+            </doc>
+          </field>
+          <field name="static_q_list" type="NX_NUMBER" units="NX_PER_LENGTH" minOccurs="0">
+            <doc>
+            1-D list of :math:`|Q|` values, 1 for each roi.
+            </doc>
+          </field>
+      </group>
+    </group>
+
+    <group type="NXsample" name="sample" minOccurs="0">
+    <!--
+      Describes the minimum requirements regarding equilibrium sample conditions. NXsample
+      permits other quantities (e.g., applied fields, crystallographic information, name, etc) that
+      may optionally be include for equilibrium conditions (which is not exclusively equilibrium
+      dynamics from XPCS analysis).
+
+      For non-equilibrium sample conditions (i.e., changing sample or process conditions
+      during the XPCS measurement) will require either a new entry or an additional atttribute.
+      -->
+      <field name="temperature_set" type="NX_NUMBER" units="NX_TEMPERATURE" minOccurs="0">
+        <doc>
+          Sample temperature setpoint, (C or K).
+        </doc>
+      </field>
+      <field name="temperature" type="NX_NUMBER" units="NX_TEMPERATURE" minOccurs="0">
+        <doc>
+          Sample temperature actual, (C or K).
+        </doc>
+      </field>
+      <group type="NXpositioner" name="position_x" minOccurs="0" />
+      <group type="NXpositioner" name="position_y" minOccurs="0" />
+      <group type="NXpositioner" name="position_z" minOccurs="0" />
+    </group>
+
+    <group type="NXnote" name="ROI" minOccurs="0" maxOccurs="unbounded">
+      <doc>
+        **THIS FIELD IS SCHEDULED FOR DELETION on or about March 1, 2022.** Contact abarbour@bnl.gov or jemian@anl.gov to object.
+
+        Region(s) of interest.
+
+        NAME: The NeXus convention, to use all upper case
+        to indicate the name (here ``roi``), is left to the file
+        writer.  In our case, follow the suggested name
+        pattern and sequence: roi_1, roi_2, roi_3, ...
+        Start with ``roi_1`` if the first one, otherwise
+        pick the next number in this sequence.
+      </doc>
```

Comment:
The suggested naming scheme could be used to indicate the sequence of a set of notes that would compete with the `sequence_index` field of NXnote and might create confusion about which sequence has priority.

Reply:
Instead, there is a comment to remove this field ~2022-03-01.  Removing this now.